go/cmd/dolt/commands/reset.go

```diff
@@ -35,23 +35,27 @@ const (
 )
 
 var resetDocContent = cli.CommandDocumentationContent{
-	ShortDesc: "Resets staged tables to their HEAD state",
-	LongDesc: "Sets the state of a table in the staging area to be that table's value at HEAD\n\n" +
-		"{{.EmphasisLeft}}dolt reset <tables>...{{.EmphasisRight}}" +
+	ShortDesc: "Resets staged or working tables to HEAD or a specified commit",
+	LongDesc: "{{.EmphasisLeft}}dolt reset <tables>...{{.EmphasisRight}}" +
 		"\n\n" +
-		"This form resets the values for all staged {{.LessThan}}tables{{.GreaterThan}} to their values at {{.EmphasisLeft}}HEAD{{.EmphasisRight}}. " +
+		"The default form resets the values for all staged {{.LessThan}}tables{{.GreaterThan}} to their values at {{.EmphasisLeft}}HEAD{{.EmphasisRight}}. " +
 		"It does not affect the working tree or the current branch." +
 		"\n\n" +
 		"This means that {{.EmphasisLeft}}dolt reset <tables>{{.EmphasisRight}} is the opposite of {{.EmphasisLeft}}dolt add <tables>{{.EmphasisRight}}." +
 		"\n\n" +
 		"After running {{.EmphasisLeft}}dolt reset <tables>{{.EmphasisRight}} to update the staged tables, you can use {{.EmphasisLeft}}dolt checkout{{.EmphasisRight}} to check the contents out of the staged tables to the working tables." +
 		"\n\n" +
+		"{{.EmphasisLeft}}dolt reset [--hard | --soft] <commit>{{.EmphasisRight}}" +
```

Comment:
Should we add some description of the `hard` versus `soft` option?

Reply:
![image](https://user-images.githubusercontent.com/193449/167895709-810ac136-6f80-4058-9da5-bc6ba360d3e8.png)

_(I couldn't resist a Pet Shop Boy reference)_

Reply:
Yes plz